x-pack/plugins/apm/server/routes/transactions.ts

```diff
@@ -237,23 +238,21 @@ const transactionChartsDistributionRoute = createApmServerRoute({
       transactionName,
       transactionId = '',
       traceId = '',
```

Comment:
Why are these defaulting to empty? It seems like a way to trick the compiler to think they are required, when in fact they are optional.

Reply:
That code didn't change in this PR, I know too little about the whole setup to tell and would prefer to keep as is in this PR. https://github.com/elastic/kibana/blame/332c742faf2afd23fe8f1cbfc3f61df6e355e5c8/x-pack/plugins/apm/server/routes/transactions.ts#L239